### PR TITLE
Parse NumberInput

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: lona/html/data_binding/inputs.py [hunk 1]
```diff
@@ -173,19 +173,18 @@ def __init__(
         )
         self.min = min
         self.max = max
```
Comment (fscherf) [inline lona/html/data_binding/inputs.py]: Shouldn't we do the same for these to values as well?

Comment (fscherf) [reply lona/html/data_binding/inputs.py]: Oh! That's done in the other commit.

**Review by fscherf — CHANGES_REQUESTED** [review]
Could you add more documentation to the patch descriptions? These patches seem to do multiple things. A sentence on what is broken would help too

**Comment by maratori** [comment]
@fscherf You are right. I messed all in one PR. Will create another.

**Pull request closed without merging**